lib/finicity/client.rb

```diff
@@ -147,8 +147,8 @@ def authenticate!
       end
     end
 
-    def delete_account(customer_id, account_id)
-      request = ::Finicity::V1::Request::DeleteAccount.new(token, customer_id, account_id)
+    def delete_account(customer_id, institution_id)
```

Comment:
Maybe should just add a new method, something like `delete_accounts_by_institution_id` and use that.